### DISCUSSION TITLE
Tapping on the notification results in a 404 error.

### DISCUSSION THREAD

**Discussion by tmorio** [mastodon/mastodon#23380] [Q&A] [answered]
Updated from v3.5.5 to v4.0.0 to v4.0.2.
After the update, the following error message appears when tapping the notification that arrives at the PWA.

`The page you are looking for isn't here.`

The server log is as follows
`<DateTime> <HOSTNAME> bundle[<PID>]: [<UUID>] method=GET path=/@<responderUserName>/<ID> format=html controller=StatusesController action=show status=404 duration=18.95 view=1.93 db=4.09`

The relevant post has not been deleted and the notification can be seen from the web. (The URL sent in the notification is 404.)

How can I resolve this?

Additional information.
The notification from the local user is now 404.
However, notifications from remote users can be opened successfully.

**Comment by ClearlyClaire** [comment, marked as answer]
What event lead to the notification? Is the `@<responderUserName>` part pointing to the wrong user? It sounds like it could be https://github.com/mastodon/mastodon/pull/23286

**Reply by tmorio** [reply]
Thank you for your response.
The event occurred as described in this PR.
